avionics/tests/fake_include/state_input_struct.h

```diff
@@ -0,0 +1,11 @@
+#ifndef STATE_INPUT_STRUCT_HPP_4A01F791ECBA4E3CB8BC12436C41ABC9
+#define STATE_INPUT_STRUCT_HPP_4A01F791ECBA4E3CB8BC12436C41ABC9
+
+struct StateInput {
```

Comment:
sorry for forgetting to update you - states now directly take a Calculator instance/reference; it was kind of awkward to have this intermediate structure, since Calculator already needed to keep track of the current physical state (for use in estimation).